packages/visx-shape/src/shapes/Polygon.tsx

```diff
@@ -39,10 +39,12 @@ export const getPoints = ({
 
 export type PolygonProps = {
   /** Number of polygon sides. */
-  sides: number;
+  sides?: number;
   /** Size of the shape. */
-  size: number;
+  size?: number;
   /** className to apply to polygon element. */
+  /** Points to use to render the polygon. If this is defined, `sides`, `size`, `rotate`, and `center` are ignored. */
```

Comment:
looks like this new prop + doc string got inserted between the `className` doc string and the `className` prop, can you co-locate those again?

Reply:
Yes!